File: src/main/java/org/dukcode/ps/codetree/trail2/IntroCreateARectangleUsingAFunction.java
package org.dukcode.ps.codetree.trail2;

import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.StringTokenizer;

/**
 * @see <a
 * href="https://www.codetree.ai/ko/trails/complete/curated-cards/intro-create-a-rectangle-using-a-function/description">LINK</a>
 */
public class IntroCreateARectangleUsingAFunction {

  private static final String CONTENT = "1";

  private static BufferedReader br;
  private static BufferedWriter bw;
  private static StringTokenizer st;

  private static int h;
  private static int w;

  public static void main(String[] args) throws IOException {
    br = new BufferedReader(new InputStreamReader(System.in));
    bw = new BufferedWriter(new OutputStreamWriter(System.out));

    st = new StringTokenizer(br.readLine());

    h = Integer.parseInt(st.nextToken());
    w = Integer.parseInt(st.nextToken());

    printSquare(h, w);

    br.close();
    bw.close();

  }

  private static void printSquare(int h, int w) throws IOException {
    for (int y = 0; y < h; y++) {
      for (int x = 0; x < w; x++) {
        bw.write(CONTENT);
      }
      bw.newLine();
    }
    bw.newLine();
  }
}

File: src/main/kotlin/org/dukcode/ps/codetree/trail02/chapter01/lesson01/IntroCreateARectangleUsingAFunction.kt
package org.dukcode.ps.codetree.trail02.chapter01.lesson01

import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.util.StringTokenizer

private val br = BufferedReader(InputStreamReader(System.`in`))
private val bw = BufferedWriter(OutputStreamWriter(System.out))

/**
 * @see [LINK](https://www.codetree.ai/ko/trails/complete/curated-cards/intro-create-a-rectangle-using-a-function/description)
 */
fun main() {
    val st = StringTokenizer(br.readLine())
    val h = st.nextToken().toInt()
    val w = st.nextToken().toInt()

    printNums(h, w)

    bw.flush()
}

fun printNums(
    h: Int,
    w: Int,
) {
    repeat(h) {
        repeat(w) {
            bw.write('1'.code)
        }
        bw.newLine()
    }
}
